staxapp/config.py

```diff
@@ -7,11 +7,11 @@
 import staxapp
 from staxapp.exceptions import ApiException
 
-logging.getLogger().setLevel(logging.DEBUG)
-logging.getLogger("boto3").setLevel(logging.WARNING)
-logging.getLogger("botocore").setLevel(logging.WARNING)
-logging.getLogger("nose").setLevel(logging.WARNING)
-logging.getLogger("urllib3").setLevel(logging.WARNING)
+logging.getLogger().setLevel(os.getenv("LOG_LEVEL", logging.DEBUG))
```

Comment:
```suggestion
logging.getLogger().setLevel(os.getenv("LOGLEVEL", logging.INFO))
```

Reply:
> 

Does the existing convention in this file not lend itself more to `LOG_LEVEL`? (ie STAX_REGION, API_VERSION)

Reply:
That's fair as well - I'll happily accept either (the requested change to INFO aside) 

The suggestion for LOGLEVEL was to fit what I could see libraries / projects using in the absence of a _convention_ (I had a bit of a look around but nothing exhaustive) but it was not a strongly held position

Reply:
Thanks @Anton0. We internally had some debate a few years ago about this same thing - `LOG_LEVEL` vs `LOGLEVEL`.

RE: the changing of boto3/botocore, I think it's reasonable to keep the `WARNING`. It was the "ALL" loggers to default which I thought was a bit too verbose.